### PR TITLE
Add more information on default limit number of search results

### LINKED ISSUE #1803: Provide more information about search results
**Is your feature request related to a problem? Please describe.**

Currently if no result, the terminal doesn't says so, it also doesn't say it only lists 10 results by default

**Describe the solution you'd like**

Return no results when search/wsearch got 0 results, also say that only listing 10 results by default

### PR DESCRIPTION
**Pull request by yshen-google**
Due to #2191, it seems travis-ci is not run on PRs from forked repo. Thus, create this PR from branch on the current repo.

1. Print out the default limit on search results
2. Print out no results found if so

### CODE REVIEW

File: codalab/lib/bundle_cli.py [hunk 1]
```diff
@@ -2040,6 +2046,14 @@ def get(i, info, col):
                 else:
                     return info.get(col, nested_dict_get(info, 'metadata', col))
 
+            if len(bundle_info_list) == bundle_model.SEARCH_RESULTS_LIMIT:
+                print(
+                    'Only {} results are shown. Use .limit=N to show N.'.format(
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: "show the first N"

File: codalab/lib/bundle_cli.py [hunk 2]
```diff
@@ -3184,18 +3198,28 @@ def do_wls_command(self, args):
             'worksheets',
             params={'keywords': args.keywords, 'include': ['owner', 'group_permissions']},
         )
+
         if args.uuid_only:
             for row in worksheet_dicts:
                 print(row['uuid'], file=self.stdout)
         else:
             if worksheet_dicts:
+                if len(worksheet_dicts) == bundle_model.SEARCH_RESULTS_LIMIT:
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Could you factor this block out to avoid code duplication?

File: codalab/lib/bundle_cli.py (hunk 1 above)
Comment (yshen-google) [reply codalab/lib/bundle_cli.py]: Done.

File: codalab/lib/bundle_cli.py (hunk 2 above)
Comment (yshen-google) [reply codalab/lib/bundle_cli.py]: Sounds good.

**Review by percyliang — APPROVED** [review]
Great, thanks!

**Review by candicegjing — APPROVED** [review]
Looks great to me!

**Pull request merged**